SmartBadge/pong.py

```diff
@@ -16,6 +16,8 @@ def __init__(self):
         self.direction = [1, 1]
 
     def reset(self):
+        self.x = 16
+        self.y = 16
```

Comment:
Why 16?
Could it be made a global constant?

Reply:
As it stands, these values are all hard coded as we have not fully decided the aspect ratio for the grid

Reply:
Could we at least have them as global constants, so they can be easily changed? The 16 on its own isn't very descriptive.

Reply:
So, I'd like to tidy this up across all the files and I think this should be done after a merge to master